imap/httpd.c

```diff
@@ -1383,6 +1383,11 @@ static int reset_saslconn(sasl_conn_t **conn)
 }
 
 
+static const char *http_versions[] = {
+    "HTTP/1.0", HTTP_VERSION, HTTP2_VERSION
+};
+
```

Comment:
If you add something like this here...

```suggestion
static const char *http_versions[] = {
    "HTTP/1.0", HTTP_VERSION, HTTP2_VERSION
};
static const size_t n_http_versions = sizeof(http_versions) / sizeof(http_versions[0]);
```

Reply:
In addition, you might want to change
`static const char *http_versions[]`
to
`static const char * const http_versions[]`

Reply:
done